extensions/interactions/PencilCodeEditor/directives/PencilCodeEditor.js

```diff
@@ -21,12 +21,14 @@
  */
 
 oppia.directive('oppiaInteractivePencilCodeEditor', [
-  'HtmlEscaperService', 'UrlInterpolationService',
-  function(HtmlEscaperService, UrlInterpolationService) {
+  'HtmlEscaperService', 'UrlInterpolationService', 'EVENT_NEW_CARD_AVAILABLE',
+  function(HtmlEscaperService, UrlInterpolationService,
```

Comment:
Break after '('. Also, match line breaks in non-stringified deps to the line breaks in the stringified deps above, so that it's easy to verify that they are the same. Otherwise bad things happen during minification.

Reply:
Ok, done